postgres-appliance/build_scripts/compress_build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -ex
+
+apt-get update
+apt-get install -y busybox xz-utils
+apt-get clean
+
+rm -rf /var/lib/apt/lists/* /var/cache/debconf/* /usr/share/doc /usr/share/man /etc/rc?.d /etc/systemd
+ln -snf busybox /bin/sh
+
+files="/bin/sh"
+arch=$(uname -m)
+darch=$(uname -m | sed 's/_/-/')
+libs="$(ldd $files | awk '{print $3;}' | grep '^/' \
+    | sort -u) /lib/ld-linux-$darch.so.* /lib/$arch-linux-gnu/ld-linux-$darch.so.* /lib/$arch-linux-gnu/libnsl.so.* \
+    /lib/$arch-linux-gnu/libnss_compat.so.*"
+
+(echo /var/run /var/spool "$files" "$libs" | tr ' ' '\n' && realpath "$files" "$libs") | sort -u | sed 's/^\///' > /exclude
```

Comment:
Quoted "$libs" disables name globbing, but we rely on it.

```suggestion
libs=("$(ldd $files | awk '{print $3;}' | grep '^/' \
    | sort -u)" /lib/ld-linux-$darch.so.* /lib/$arch-linux-gnu/ld-linux-$darch.so.* /lib/$arch-linux-gnu/libnsl.so.* \
    /lib/$arch-linux-gnu/libnss_compat.so.*)

(echo /var/run /var/spool "$files" "${libs[@]}" | tr ' ' '\n' && realpath "$files" "${libs[@]}") | sort -u | sed 's/^\///' > /exclude
```